e2e/features/case-progression.feature

```diff
@@ -1173,7 +1174,7 @@ Feature: Case progression
 
     #    When I list the case
     When I click the `Overview` tab
-    And I click the `List the case` link
+    And I click the `list the case here` link
     Then I am on the `List the case` page
```

Comment:
I think we need to make changes the list case scenarios for example _list-case-after-submit-hearing-requirements.feature_ and  _list-case.feature_

Reply:
list-case.feature is the old flow and should remain as it is. I will change the list-case-after-submit-hearing-requirements.feature to accommodate new changes.